### DISCUSSION TITLE
Cant disable dark mode!

### DISCUSSION THREAD

**Discussion by dila1001** [saadeghi/daisyui#2825] [Q&A] [answered]
How do i disable dark mode?? For some reason, on safari (desktop), safari (ios) and chrome (ios), my app shows the 'dark' theme but i've excluded it in my config. I've even added `data-theme="lemonade"` to the root html to no avail. When i switch my device to light mode, these browsers shows the default 'light' theme instead of 'lemonade'. The only browser that works as expected is chrome (desktop). Why is that?
```js
`/** @type {import('tailwindcss').Config} */
import daisyui from 'daisyui';
import * as daisyuiThemes from 'daisyui/src/theming/themes';

export default {
	content: ['./index.html', './src/**/*.{js,ts,jsx,tsx}'],
	theme: {
		extend: {},
	},
	darkMode: 'false',
	plugins: [daisyui],
	daisyui: {
		themes: [
			{
				lemonade: {
					...daisyuiThemes['lemonade'],
					primary: '#7E9674',
					error: 'dd7373',
					// accent: '#eac95b',
					// success: '#C5D2CE',
					// secondary: '# add1B966',
				},
			},
		], // false: only light + dark | true: all themes | array: specific themes like this ["light", "dark", "cupcake"]
		darkTheme: 'false', // name of one of the included themes for dark mode
		base: true, // applies background color and foreground color for root element by default
		styled: true, // include daisyUI colors and design decisions for all components
		utils: true, // adds responsive and modifier utility classes
		prefix: '', // prefix for daisyUI classnames (components, modifiers and responsive class names. Not colors)
		logs: true, // Shows info about daisyUI version and used config in the console when building your CSS
		themeRoot: ':root', // The element that receives theme color CSS variables
	},
};
```

**Comment by saadeghi** [comment]
Can you provide a reproduction repo?

**Reply by dila1001** [reply]
Here's my repo https://github.com/dila1001/h2oasis

EDIT: I updated my iOS and now its showing lemonade!

**Reply by saadeghi** [reply, marked as answer]
> I updated my iOS

Starting from daisyUI 4.x we're using the new wide-gamut OKLCH colors.  
OKLCH colors are supported in iOS 15.4+. Older browsers have a fallback to show the default light or dark theme so for that reason it was showing the default theme.

**Reply by dila1001** [reply]
Thank you so much. This explains it!
